frontend/packages/ui-workflow/src/styles/workflow.css

```
@@ -0,0 +1,659 @@
+  --apme-yaml-header-bg: var(--pf-t--global--background--color--secondary--default);
+  --apme-yaml-comment: #6e7781;
+  --apme-yaml-punctuation: #24292f;
+  --apme-yaml-key: #0550ae;
+  --apme-yaml-string: #0a3069;
+  --apme-yaml-boolean: #0550ae;
+  --apme-yaml-number: #0550ae;
+  --apme-yaml-keyword: #cf222e;
+  --apme-yaml-important: #953800;
+  --apme-yaml-remove-bg: rgba(211, 47, 47, 0.12);
+  --apme-yaml-add-bg: rgba(26, 127, 55, 0.12);
+  background: var(--apme-yaml-bg);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)